pal/audio/minigame/player-minigame.ts

```diff
@@ -58,6 +58,8 @@ export class OneShotAudioMinigame {
         });
         const endCallback = (): void => {
             if (this._innerAudioContext) {
+                game.off(Game.EVENT_PAUSE, this._onInterruptedBegin, this);
+                game.off(Game.EVENT_RESUME, this._onInterruptedEnd, this);
```

Comment:
`game` module is in the framework layer， `pal` should not rely on it 
please use systemInfo.on('show') systemInfo.on('hide') instead